arches_lingo/src/arches_lingo/pages/SchemeList.vue

```diff
@@ -1 +1,13 @@
-<template>Schemes list placeholder</template>
+<script setup lang="ts">
+import ProgressSpinner from "primevue/progressspinner";
+import SchemeBoxes from "@/arches_lingo/components/schemes/SchemeBoxes.vue";
+</script>
+
+<template>
+    <Suspense>
+        <SchemeBoxes />
+        <template #fallback>
```

Comment:
I don't love this pattern on the page level, as it lends itself to page loader logic which I'm trying to get away from. In the future I could see having a custom error page as a fallback but for now I'd say skip this pattern.